### PR TITLE
Pud 62 Add details to Letter to Probation

### PR DESCRIPTION
**Pull request by mikedunfordEE**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/service/RecallSummaryGenerator.kt [hunk 1]
```diff
@@ -16,15 +16,15 @@ import java.time.LocalDate
...
+import java.util.Locale.ENGLISH
 
 @Component
 class RecallSummaryGenerator(
   @Autowired private val templateEngine: SpringTemplateEngine,
   @Autowired private val clock: Clock
 ) {
 
-  private val dateFormatter = DateTimeFormatter.ofPattern("dd MMM yyyy", Locale.ENGLISH)
+  private val dateFormatter = DateTimeFormatter.ofPattern("dd MMM yyyy", ENGLISH)
```
Comment (AndyNewton) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/service/RecallSummaryGenerator.kt:27]: Can supersede with your new `.asStandardDate`?
Also, interesting use of a locale here and not elsewhere??

File: src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/service/RecallSummaryGenerator.kt [hunk 2]
```diff
@@ -16,15 +16,15 @@ import java.time.LocalDate
...
   private val timeFormatter = DateTimeFormatter.ofPattern("HH:mm")
```
Comment (AndyNewton) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/service/RecallSummaryGenerator.kt:28]: Ditto `asStandardTime`?

**Review by AndyNewton — APPROVED** [review]
Approved with two minor QQs - can be for following PR.

File: src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/service/RecallSummaryGenerator.kt (hunk 1 above)
Comment (mikedunfordEE) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/service/RecallSummaryGenerator.kt:27]: Yes I intend to reuse all over the place.  Just didn't want to change too much while others were still working in this area.

Without being specific about the Locale I was getting differences locally and in circleCI e.g. Sep vs Sept

**Pull request merged**